### ISSUE TITLE
Install `imagemagick` and `php5-imagick`for use with Drupal

### ISSUE THREAD

**Issue by mccrodp** [RobLoach/Dockie#41]
Hi Rob,

I noticed that these modules were missing in Dockie when I bumped into this issue on all of my docker environments using the docker-compose-drupal setup: https://github.com/RobLoach/docker-compose-drupal/issues/6

Would it be possible to get these into the Dockie image? Thanks!

**Comment by RobLoach** [comment]
Seem like good additions to have. Think just the apt-gets do it?

**Comment by mccrodp** [comment]
Yep, apt-gets get them alright. Didn't solve my linked issue unfortunately, but surely a factor!

**Comment by mccrodp** [comment]
`apt-get install postfix` - another good one :)
